kernel/arch/dreamcast/include/dc/vmufs.h

```diff
@@ -2,6 +2,7 @@
 
    dc/vmufs.h
    Copyright (C) 2003 Megan Potter
+   Copyright (C) 2023 Falco Girgis
```

Comment:
2024

Reply:
Doooooh. Yeah this branch was from last year. XD